peachjam/js/components/ListFacets.vue

```diff
@@ -314,55 +42,105 @@ export default {
       default: () => []
     }
   },
+
   data: () => {
```

Comment:
change data to `data ()` then use the method `getFacets` set `facets`, so like this
```
data () {
    return {
      loading: false,
      facets: this.getFacets()
    };
  },
```